packages/@aws-cdk/cdk/test/test.app.ts

```diff
@@ -4,7 +4,7 @@ import { Construct, Resource, Stack, StackProps } from '../lib';
 import { App } from '../lib/app';
 
 function withApp(context: { [key: string]: any } | undefined, block: (app: App) => void): cxapi.SynthesizeResponse {
-  const app = new App(context);
+  const app = new App({ context, autoRun: false });
```

Comment:
Make `autoRun: false` the default when running outside the context of the toolkit (e.g. CDK_OUT is not set)

Reply:
`autoRun` false is not needed now i guess